x/ecocredit/server/marketplace/prune.go

```diff
@@ -17,13 +17,15 @@ import (
 func (k Keeper) PruneSellOrders(ctx context.Context) error {
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 
-	min, blockTime := timestamppb.New(time.Unix(0, 0)), timestamppb.New(sdkCtx.BlockTime())
+	// we set the min to 1 ns because nil expirations are encoded as the 0 value timestamp,
+	// and we DO NOT want those to be deleted/unescrowed.
```

Comment:
```suggestion
	// we set the min to 1 ns because nil expirations are encoded as the 0 value timestamp,
	// and we DO NOT want those to be deleted/unescrowed.
	// https://github.com/cosmos/cosmos-sdk/issues/11980
```